ethcore/src/client/client.rs

```diff
@@ -901,30 +901,30 @@ impl Client {
 				author: header.author(),
 				timestamp: header.timestamp(),
 				difficulty: header.difficulty(),
-				last_hashes: self.build_last_hashes(&header.parent_hash()),
+				last_hashes: self.build_last_hashes(header.parent_hash()),
 				gas_used: U256::default(),
 				gas_limit: header.gas_limit(),
 			}
 		})
 	}
 
-	fn build_last_hashes(&self, parent_hash: &H256) -> Arc<LastHashes> {
+	fn build_last_hashes(&self, parent_hash: H256) -> Arc<LastHashes> {
```

Comment:
this was changed was motivated to make it more clear that `parent_hash` is copied/cloned by `build_last_hashes`, i.e, pass to by value instead of cloning it inside the function.